src/html-imports.js

```diff
@@ -498,29 +509,22 @@
         // Check if was already moved to head, to handle the case where the element
         // has already been moved but it is still loading.
         if (needsMove && s.parentNode !== document.head) {
-          let rootImport = importForElement(s);
-          while (rootImport && importForElement(rootImport)) {
-            rootImport = importForElement(rootImport);
-          }
           // Replace the element we're about to move with a placeholder.
-          // NOTE: we first have to append the element to the new parent, then
-          // we can put the placeholder at its place, otherwise load/error events
-          // seem to be fired too early.
```

Comment:
Yeah this comment is now wrong as we changed the order of which things happen: now we first append the placeholder by relying on nextSibling (which won't move).